### PR TITLE
Update all the text widgets having the same name with the same value

### LINKED ISSUE #13271: Text widgets with the same name must share the same value
Attach (recommended) or Link to PDF file here:
[multiple.pdf](https://github.com/mozilla/pdf.js/files/6344777/multiple.pdf)

Steps to reproduce the problem:
1. Write something in a field

What is the expected behavior? (add screenshot)
The 3 fields must have the same value

What went wrong? (add screenshot)
The 2 other fields are empty

The three fields share the same name `Text3`.
It works as expected in Adobe and Chrome.

### PR DESCRIPTION
**Pull request by calixteman**
Fixes #13271.

### CODE REVIEW

File: test/integration/annotation_spec.js [hunk 1]
```diff
@@ -87,3 +87,38 @@ describe("Checkbox annotation", () => {
...
+
+    beforeAll(async () => {
+      pages = await loadAndWait("issue13271.pdf", "[data-annotation-id='24R']");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must all the fields with the same value", async () => {
```
Comment (Snuffleupagus) [inline test/integration/annotation_spec.js:103]: This reads a little bit weird, how about "all fields must have the same value" instead?

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Thank you for improving this!

**Pull request merged**